### PR TITLE
aruha-932: check admin permission when delete et

### PR DESCRIPTION
**Pull request by adyach**
aruha-932

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/EventTypeService.java [hunk 1]
```diff
@@ -168,6 +169,9 @@ public void delete(final String eventTypeName, final Client client)
             if (!client.idMatches(eventType.getOwningApplication())) {
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/service/EventTypeService.java]: Why you left this thing here?

Comment (antban) [inline src/main/java/org/zalando/nakadi/service/EventTypeService.java]: Oh, yeah, this check is constantly turned off, and we will remove it later.

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**